tests/python/unittest/test_runtime_micro.py

```diff
@@ -173,7 +234,7 @@ def test_resnet_random():
                                        resnet_func.body.args[0],
                                        resnet_func.ret_type)
 
-    with micro.Session(DEVICE_TYPE, TOOLCHAIN_PREFIX):
+    with micro.create_session(DEVICE_TYPE, TOOLCHAIN_PREFIX):
```

Comment:
Session is fine as it is a constructor

Reply:
oh. I was just trying to mirror the graph runtime. I'll change it back.